awx/main/tasks/jobs.py

```diff
@@ -1873,6 +1873,11 @@ def build_args(self, system_job, private_data_dir, passwords):
             if system_job.job_type in ('cleanup_jobs', 'cleanup_activitystream'):
                 if 'days' in json_vars:
                     args.extend(['--days', str(json_vars.get('days', 60))])
+                if 'batch_size' in json_vars:
+                    default_value = 100000
+                    if system_job.job_type == 'cleanup_activitystream':
+                        default_value = 500
+                    args.extend(['--batch-size', str(json_vars.get('batch_size', default_value))])
```

Comment:
```suggestion
                    args.extend(['--batch-size', str(json_vars['batch_size'])])
```

By this point you already established that "batch_size" is in the vars, so there's no need to process default values in this location.

Reply:
Thanks for code review.